e2e/tests/init_test.go

```diff
@@ -31,12 +33,21 @@ var (
 	accNameToAppMap      = make(map[string]apptypes.Application)
 	accNameToSupplierMap = make(map[string]sharedtypes.Supplier)
 
-	keyRingFlag = "--keyring-backend=test"
+	featuresPathFlag string
+	keyRingFlag      = "--keyring-backend=test"
```

Comment:
flagKeyRingDefault?

Reply:
I'm reluctant to align the naming between these two "flag vars". This is categorically different than the features path flag, it is not registered as a go flag but is instead both the key and value of a flag which is passed to binaries called by the go command API.